hermes/src/store.rs

```diff
@@ -265,10 +278,24 @@ impl Store {
             .iter()
             .map(|message_state| match message_state.message {
                 Message::PriceFeedMessage(price_feed) => Ok(PriceFeedUpdate {
```

Comment:
`impl From<PriceFeedMessage> for PriceFeed` ?

Reply:
It's not on this package. Do you think it's good to add pyth-sdk dependency in pythnet sdk? 🤔

Reply:
Could also do `impl Into<PriceFeed> for PriceFeedMessage`, but It's no big deal.